com.ibm.streamsx.slack/impl/java/src/com/ibm/streamsx/slack/SendSlackMessage.java

```diff
@@ -165,30 +170,32 @@ public void process(StreamingInput<Tuple> stream, Tuple tuple)
     		}
     	} catch (Exception e) {
     		_trace.error(e);
-    		return;
+    		return true;
     	}
     	
     	// Send Slack message if slack webhook URL is specified.
 		if (slackUrl != null) {
+			StringEntity params = new StringEntity("{\"text\" : \"" + message + "\""
```

Comment:
Rather than manually create serialized JSON an `JSONObject` should be created and then serialized.

The existing approach I think will break if the message contains a double quote. Which then also allows "injection" of any arbitrary JSON into the request.